components-java-sdk-examples/src/main/java/io/dapr/examples/statestore/InMemoryStateStore.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dapr.examples.statestore;
+
+
+import io.dapr.components.domain.state.DeleteRequest;
+import io.dapr.components.domain.state.Exceptions;
+import io.dapr.components.domain.state.GetRequest;
+import io.dapr.components.domain.state.GetResponse;
+import io.dapr.components.domain.state.SetRequest;
+import io.dapr.components.domain.state.StateStore;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class InMemoryStateStore implements StateStore {
+
+  private final Map<String, GetResponse> entries = new HashMap<>();
+
+  @Override
+  public Mono<Void> init(Map<String, String> properties) {
+    // log.info("Initializing InMemoryState Store."); // I was born ready, babe
+    return Mono.empty();
+  }
+
+  //  @Override
+  //  public Mono<List<String>> getFeatures() {
+  //    return Mono.just(Collections.emptyList());
+  //  }
```

Comment:
This can be removed. I thought about advertising etag support but I don't have it done properly,